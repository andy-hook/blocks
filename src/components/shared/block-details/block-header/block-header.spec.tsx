import React from "react"
import BlockHeader from "./block-header"
import { renderWithTheme } from "@test-utils"

describe("<BlockHeader />", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme("dark", <BlockHeader />)
    expect(tree).toMatchSnapshot()
  })
})
