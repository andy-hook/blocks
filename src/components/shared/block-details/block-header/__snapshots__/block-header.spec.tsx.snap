// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlockHeader /> renders correctly 1`] = `
<header
  className="sc-bZQynM leOjQD"
>
  <div
    className="sc-htoDjs jzmsjH"
  >
    <div
      className="sc-bdVaJa jlBogh sc-gzVnrw hBjlQU"
      size="lg"
    >
      #Â 
    </div>
    <div
      className="sc-htpNat iflsxI"
      size="lg"
    >
       Transactions
    </div>
  </div>
  <a
    className="sc-ifAKCX gyAyBF"
    href="https://www.google.com"
    type="primary"
  >
    <span
      className="sc-EHOje dkmHZI"
    >
      View transactions
    </span>
  </a>
</header>
`;
